astra/src/test/java/com/slack/astra/server/BulkIngestApiTest.java

```diff
@@ -66,6 +69,23 @@ public class BulkIngestApiTest {
 
   private static String DOWNSTREAM_TOPIC = "test-topic-out";
 
+  private static boolean useKafkaTransactions;
+
+  @BeforeAll
+  public static void useTransactions() {
+    Random random = new Random();
```

Comment:
This is the first time we'll be using this sort of paradigm. i.e depending on the test run one code path is executed. 

The other option is to have the `testDocumentInKafkaSimple`  test twice. Thoughts @bryanlb ?

Reply:
I like the idea of testing it twice personally